docs/source/en/api/utilities.mdx

```diff
@@ -0,0 +1,23 @@
+# Utilities
+
+Utility and helper functions for working with 🤗 Diffusers.
+
+## randn_tensor
+
+[[autodoc]] diffusers.utils.randn_tensor
+
+## numpy_to_pil
+
+[[autodoc]] utils.pil_utils.numpy_to_pil
+
+## pt_to_pil
+
+[[autodoc]] utils.pil_utils.pt_to_pil
```

Comment:
Some of these are usually accessed as static functions of the pipelines, which is usually convenient. Not sure if we should cross-reference them somehow (not necessary for this PR, just to keep in mind).